### PR TITLE
feat: adapt CLI to pass data instead of queries

### PR DESCRIPTION
**Pull request by m-pizarro**
## Issue tracker links

_Add links to any relevant tasks/stories/bugs/pagerduty/etc_

*Example - dummy TODO project*

[TODO-123](https://autoclouddev.atlassian.net/browse/TODO-123)

## Changes/solution

_How does this change address the problem?_

## Testing

_Describe how the testing was done, plus evidence, if not covered by automated tests_

## Notes and considerations

_Add any additional notes and/or considerations_

## Dependencies

_Add dependencies on any other PRs, if applicable

### CODE REVIEW

File: src/plugins/policyPack/index.ts [hunk 1]
```diff
@@ -137,10 +137,12 @@ export default class PolicyPackPlugin extends Plugin {
   }
 
   private async executeRule({
+    data,
     rules,
     policyPack,
     storageEngine,
```
Comment (aacotroneo) [inline src/plugins/policyPack/index.ts]: unused, we should remove

File: src/plugins/policyPack/index.ts [hunk 2]
```diff
@@ -159,18 +161,17 @@ export default class PolicyPackPlugin extends Plugin {
...
               rules: subRules,
               policyPack,
               storageEngine,
             }))
           )
         } else {
-          const { data } = rule.gql
-            ? await storageEngine.query(rule.gql)
-            : { data: undefined }
+          const ruleData = rule.gql ? data : undefined
```
Comment (aacotroneo) [inline src/plugins/policyPack/index.ts:169]: this is unneeded too... ruleData is data (the whole data)

File: src/plugins/policyPack/index.ts [hunk 3]
```diff
@@ -274,13 +275,69 @@ export default class PolicyPackPlugin extends Plugin {
     }
   }
 
+  private getLinkedData({ providerData }: { providerData: ProviderData }): any {
```
Comment (aacotroneo) [inline src/plugins/policyPack/index.ts]: maybe we can move this to somewhere else in the sdk and export it (it's just generically linking data)

File: src/plugins/policyPack/index.ts (hunk 2 above)
Comment (aacotroneo) [reply src/plugins/policyPack/index.ts:169]: oh, manual rules.. lets tackle that in a future PR

File: src/plugins/policyPack/index.ts (hunk 3 above)
Comment (mfranceschit) [reply src/plugins/policyPack/index.ts]: I agree. This could be reusable in the future, we can put it under the `utils` folder

Comment (m-pizarro) [reply src/plugins/policyPack/index.ts]: done!

**Review by mfranceschit — APPROVED** [review]
LGTM

**Pull request merged**